interfaces/apparmor/template.go

```diff
@@ -192,6 +192,11 @@ var defaultTemplate = []byte(`
   /usr/bin/ r,
   /usr/share/distro-info/*.csv r,
 
```

Comment:
Currently there is no mechanism that will regenerate all apparmor profiles on snapd update. We should come up with one for 2.14